src/main/java/com/endava/soa_5/basic/Exercise_1.java

```diff
@@ -0,0 +1,32 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_1 {
```

Comment:
Add a parametrized and default constructor.

Reply:
Done!